examples/vault-variable-test/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
```

Comment:
This is needed to make a suggestion below work
```suggestion
use anyhow::{Result, Context};
```